CONFIGURATION.md

```diff
@@ -0,0 +1,128 @@
+# Configuring NTPD-rs
+
+NTPD-rs is an implementation of NTP completely written in Rust, with a focus on exposing a minimal attack surface. The project is currently in an early stage, but if you want to try out NTPD-rs on a non-critical system, this guide provides the basic information needed to properly build and configure NTPD-rs.
+
+## Limitations
+
+The current implementation has several important limitations which it is important to be aware of. First of all, the current implementation is an NTP client only, and does not support acting as an NTP server.
+
+Second, there are currently a few limitations around configuring connections to other NTP nodes:
+ - Only Client-Server connections are supported
+ - DNS lookup is currently only done at startup. Changes in the IP address 
+ - There is no support for NTP pools yet. Multiple 
+ - Changes in network interfaces are not picked up dynamically and will require a reboot of the daemon.
+
+Finally, please be aware that this is very early stage software, and is not yet ready for production software. This software SHOULD NOT be used on any system where you cannot handling experiencing issues, either reliability or security related.
+
+## Building
+
+Currently, NTPD-rs only support linux based operating systems. Our current testing only targets linux kernels after version 5.0.0, older kernels may work but are not guaranteed.
+
+NTPD-rs is written in rust, and requires cargo 1.61.0 at a minimum to be built. We strongly recommend using [rustup](https://rustup.rs) to install rust/cargo, as the version provided by system package managers tend to be out of date.
```

Comment:
Why does it need 1.61? IMO it would be good to stick to an older minimum supported Rust version if that's not too painful (and as far I know it's usually pretty trivial).

Reply:
Changed the wording to indicate 1.61.0 is what we test with. The argumentation for this is essentially the same as above for the kernel version.